opennms-services/src/main/java/org/opennms/netmgt/threshd/ThresholdingSet.java

```diff
@@ -356,10 +365,14 @@ protected boolean passedThresholdFilters(CollectionResourceWrapper resource, Thr
                 }
             }
         }
-        if (operator.equals("and") && andResult) {
+
+        if (result) {
```

Comment:
Why have you changed the logic here ? the "result" boolean makes sense ONLY when the operator was a NaN  (which is why I originally called it "andResult").

My suggestion is, restore all the original logic and only focus on adding the additional logging statements you want.